### ISSUE TITLE
NDepend v3 .project_file is not turned into a windows path, therefore fails ndepend.console.exe's validation

### ISSUE THREAD

**Issue by petemounce** [mxriverlynn/Albacore#192]
Using Albacore 0.2.7, NDepend v3.9.0.5848, on Win2008 R2 x64, and the following rakefile snippet, I consistently get the error below.  I think the NDependTask project_file parameter needs to be `.gsub('/','\\').`'d

I am able to work around the issue by redefining the `create_parameters` method like so:

```
class NDepend
  def create_parameters
    params = []
    params << File.expand_path(@project_file).gsub('/','\\') #NDepend v3 forces absolute windows-path
    return params
  end
end
```

**Comment by haf** [comment]
There must be a better way to do this then gsubbing all paths at all places.

For example, we have a mixin called SupportsLinuxEnvironment -- but why would be need to specifically support linux? Instead, why don't we place that logic in the execute_command task?

**Comment by petemounce** [comment]
I've found that some Windows executables care about the direction of slashes; others don't.  NDepend is particularly particular; it's the only one I've found that actively fails if it sees an invalid path.

**Comment by haf** [comment]
Right now, my problem is that I don't have a NCover license ready to use to test with. I've emailed them to ask for one though.

**Comment by petemounce** [comment]
If you like, I can test out a private build of the gem.

Did you mean NDepend, though, not NCover?

**Comment by haf** [comment]
Yeah, I meant NDepend. I'll post an update here when I've looked at the normalizing paths.
